app/points/calculate-available-points.js

```diff
@@ -0,0 +1,37 @@
+const DefaultContractedHours = require('./domain/default-contracted-hours')
+const OM_TYPE_IDS = require('./constants/offender-manager-type-ids')
+
+var calculateAvailablePoints = function (nominalTarget, baseHours, hoursReduction, defaultContractedHoursForBand) {
+  return (nominalTarget * (baseHours / defaultContractedHoursForBand)) * ((baseHours - hoursReduction) / baseHours)
+}
+
+module.exports = function (nominalTarget, offenderManagerTypeId, contractedHoursPerWeek,
+  hoursReduction, defaultContractedHours) {
+  if (!(defaultContractedHours instanceof DefaultContractedHours)) {
+    throw new Error('defaultContractedHours should be an instance of DefaultContractedHours')
+  }
+
+  // TODO: could maybe store this as part of the dict with the values as a hasAvailability flag?
```

Comment:
I like this idea, it would be interesting to understand why they don't have availability and then we could perhaps attach a flag that provides deeper meaning outside the context of this calculation. I wouldn't implement it right now because I don't think we have that context currently, but certainly something to come back to.